chain/client/src/stateless_validation/partial_witness/partial_witness_actor.rs

```diff
@@ -327,6 +338,30 @@ impl PartialWitnessActor {
             )));
         }
 
+        // TODO(https://github.com/near/nearcore/issues/11301): replace these direct DB accesses with messages
+        // sent to the client actor
```

Comment:
point to https://github.com/near/nearcore/commit/e186dc7c0b467294034c60758fe555c78a31ef2d about what kind of messages would be needed when addressing this TODO.

Reply:
added